spec/controllers/posts_controller_spec.rb

```diff
@@ -1,5 +1,67 @@
 require 'rails_helper'
+require 'spec_helper'
 
 RSpec.describe PostsController, type: :controller do
 
```

Comment:
I'd expect the test file to have this structure

```
RSpec.describe PostsController, type: :controller do
  describe '#show' do
    <examples go here>
  end
end
```

Then you can, if you want, have several examples for each controller action, if there are several things you want to test in isolation.